src/cml.js

```diff
@@ -140,13 +155,20 @@ class CML {
   }
 
   async commentCreate(opts = {}) {
+    const { remark } = await import('remark');
+    const { visit } = await import('unist-util-visit');
```

Comment:
please do the import statements on top

Reply:
I'm afraid that it's not possible

Reply:
ummm I do not fully understand

Reply:
Sorry for the cryptic reply. I mean that those are ESM (asynchronous) imports and the best we can do is having them at the top of an asynchronous function.

* https://github.com/iterative/cml/issues/1051

Reply:
Sorry for the cryptic reply. I mean that those are ESM (asynchronous) imports and the best we can do is having them at the top of an asynchronous function.

* https://github.com/iterative/cml/issues/1051

Reply:
This pull request is not blocked by #1051

Reply:
Babel is necessary because:
1. Jest doesn't fully support ESM yet
2. Vercel `pkg` doesn't support ESM at all, and will also have to depend on Babel; see #1051

Reply:
thats what I say. I would prefer to have included all that code in #1051 and here just the new feature

Reply:
> Why do they have to be async?

Why not in the top and async? Even if we use Babel imports should be done at the top

Reply:
again... why async?
Why do you do the load async?

Reply:
I mean

```js
const uuid = require('uuid');
const winston = require('winston');

const { remark } = import('remark');
const { visit } = import('unist-util-visit');

```

Reply:
`import` has to be `await`ed